core/src/components/alert/test/a11y/alert.e2e.ts

```diff
@@ -28,6 +28,59 @@ const testAria = async (
   expect(ariaDescribedBy).toBe(expectedAriaDescribedBy);
 };
 
+configs({ modes: ['ios'], directions: ['ltr'] }).forEach(({ title, screenshot, config }) => {
```

Comment:
Not to sound like a broken record but adding an annotation for the issue would be great.

Reply:
Sorry! I'll add this.